user/TASK/gimbal_task/gimbal_task.c

```diff
@@ -121,24 +149,50 @@ static void get_new_data(Gimbal_t *gimbal_data){
  */
 static void update_setpoints(Gimbal_t *gimbal_set){
     
-    gimbal_set->yaw_motor.pos_set += (-1) * int16_deadzone(gimbal_set->rc_update->rc.ch[2], -DEADBAND, DEADBAND) / 10;
-    gimbal_set->pitch_motor.pos_set += int16_deadzone(gimbal_set->rc_update->rc.ch[3], -DEADBAND, DEADBAND) / 10;
+    if(gimbal_set->rc_update->rc.s[0] == RC_SW_MID || gimbal_set->rc_update->rc.s[0] == RC_SW_UP){
```

Comment:
We should define LEFT_SWITCH and RIGHT_SWITCH in remote_control.h and replace the 0 in rc.s[0]

Reply:
Done